### PR TITLE
instruments: add proc_stat

### PR DESCRIPTION
**Pull request by setrofim**
Add an instrument that monitors CPU load using data from /proc/stat

### CODE REVIEW

File: wa/instruments/proc_stat/gather-load.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```
Comment (marcbonnici) [inline wa/instruments/proc_stat/gather-load.sh]: This shebang causes the script to fail on devices that do not have sh present at that path. Can this be updated to invoke via the busybox implementation?

Comment (setrofim) [reply wa/instruments/proc_stat/gather-load.sh]: Yup, done.

File: wa/instruments/proc_stat/__init__.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    description = '''
+    Collect CPU load information from /proc/stat.
+    '''
+
+    parameters = [
+        Parameter('period', int, default=5,
+                    constraint=lambda x: x > 0,
+                    description='''
+                    Time (in seconds) between collections.
+                    '''),
```
Comment (marcbonnici) [inline wa/instruments/proc_stat/__init__.py]: Could you align these lines to keep pep8 happy? 
(https://travis-ci.org/ARM-software/workload-automation/jobs/647303406)

Comment (setrofim) [reply wa/instruments/proc_stat/__init__.py]: Ah my bad. I had an old version of `flake8` on my machine that didn't pick this up until I upgraded it. Fixed now.

**Pull request merged**